mttex.sty

```diff
@@ -79,12 +79,20 @@
 \DeclareOption{techrpt}{\secc@techrpttrue}
 \DeclareOption{paper}{\secc@techrptfalse}
 
+%hyperref declaration?
```

Comment:
This is unnecessary. LaTeX has a built-in way of overriding required packaged. I've removed `draft` and added instructions on how to do this to mttex.